### ISSUE TITLE
Post targets are not executed in dry-run mode

### ISSUE THREAD

**Issue by k1llabee** [jscutlery/semver#676]
According to documentation, there is a possibility to provide a dry-run flag to targets that are set in `postTargets` parameter.
But there are never executed:
https://github.com/jscutlery/semver/blob/d6a436cfb9df0486a722f7a6642378baaaa67ef8/packages/semver/src/executors/version/index.ts#L193

What is the desired behavior?

**Comment by edbzn** [comment]
Yeah, it makes no sense, we need to revert the provided dryRun option and keep not running post targets in dryRun mode.

**Comment by bencat-sixense** [comment]
Is there any news on this ?
Thanks :)
